openapi.yaml

```
@@ -7,6 +7,10 @@ info:
     name: Train Support
     url: https://example.com/support
     email: support@example.com
+  license: 
+    name: Creative Commons Attribution Non Commercial Share Alike 4.0 International
+    identifier: CC-BY-NC-SA-4.0
```

@paulRbr @ChristopheDujarric does that match with our other licences? (sorry, I'm not the best one to review this)

---

I guess we can remove the `NC` (non commercial) as we are discussing for the content hub

```suggestion
  license: 
    name: Creative Commons Attribution Non Commercial Share Alike 4.0 International
    identifier: CC-BY-SA-4.0
```

---

This might be a larger question then as @ChristopheDujarric picked this license for the whole repo, and all of the bump-sh-examples repos. I'm just reusing that same license here. https://github.com/bump-sh-examples/train-travel-api/blob/main/LICENSE_CC-BY-NC-SA-4.0

---

Yeh we've had a discussion internally to change it : https://github.com/bump-sh/docs/pull/188

Could you adapt it here too please?

---

Ahh cool ok, I'll go update all the repos.